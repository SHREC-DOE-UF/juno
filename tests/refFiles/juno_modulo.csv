ComponentName, StatisticName, StatisticSubId, StatisticType, SimTime, Rank, Sum.u64, SumSQ.u64, Count.u64
cpu:customhandler, calls-to-rand, , Accumulator, 579213, 0, 0, 0, 0
cpu:customhandler, calls-to-rseed, , Accumulator, 579213, 0, 0, 0, 0
cpu, cycles, , Accumulator, 579213, 0, 1389, 1389, 1389
cpu, instructions, , Accumulator, 579213, 0, 1344, 1344, 1344
cpu, mem-reads, , Accumulator, 579213, 0, 3, 3, 3
cpu, mem-writes, , Accumulator, 579213, 0, 0, 0, 0
l1cache, prefetch_opportunities, , Accumulator, 579213, 0, 1, 1, 1
l1cache, prefetches_issued, , Accumulator, 579213, 0, 1, 1, 1
l1cache, prefetches_canceled_by_page_boundary, , Accumulator, 579213, 0, 0, 0, 0
l1cache, prefetches_canceled_by_history, , Accumulator, 579213, 0, 0, 0, 0
l1cache, Prefetch_requests, , Accumulator, 579213, 0, 1, 1, 1
l1cache, Prefetch_hits, , Accumulator, 579213, 0, 0, 0, 0
l1cache, Prefetch_drops, , Accumulator, 579213, 0, 0, 0, 0
l1cache, TotalEventsReceived, , Accumulator, 579213, 0, 6, 6, 6
l1cache, TotalEventsReplayed, , Accumulator, 579213, 0, 0, 0, 0
l1cache, TotalNoncacheableEventsReceived, , Accumulator, 579213, 0, 0, 0, 0
l1cache, CacheHits, , Accumulator, 579213, 0, 2, 2, 2
l1cache, GetSHit_Arrival, , Accumulator, 579213, 0, 2, 2, 2
l1cache, GetXHit_Arrival, , Accumulator, 579213, 0, 0, 0, 0
l1cache, GetSXHit_Arrival, , Accumulator, 579213, 0, 0, 0, 0
l1cache, GetSHit_Blocked, , Accumulator, 579213, 0, 0, 0, 0
l1cache, GetXHit_Blocked, , Accumulator, 579213, 0, 0, 0, 0
l1cache, GetSXHit_Blocked, , Accumulator, 579213, 0, 0, 0, 0
l1cache, CacheMisses, , Accumulator, 579213, 0, 2, 2, 2
l1cache, GetSMiss_Arrival, , Accumulator, 579213, 0, 2, 2, 2
l1cache, GetXMiss_Arrival, , Accumulator, 579213, 0, 0, 0, 0
l1cache, GetSXMiss_Arrival, , Accumulator, 579213, 0, 0, 0, 0
l1cache, GetSMiss_Blocked, , Accumulator, 579213, 0, 0, 0, 0
l1cache, GetXMiss_Blocked, , Accumulator, 579213, 0, 0, 0, 0
l1cache, GetSXMiss_Blocked, , Accumulator, 579213, 0, 0, 0, 0
l1cache, GetS_recv, , Accumulator, 579213, 0, 4, 4, 4
l1cache, GetX_recv, , Accumulator, 579213, 0, 0, 0, 0
l1cache, GetSX_recv, , Accumulator, 579213, 0, 0, 0, 0
l1cache, GetSResp_recv, , Accumulator, 579213, 0, 0, 0, 0
l1cache, GetXResp_recv, , Accumulator, 579213, 0, 2, 2, 2
l1cache, PutS_recv, , Accumulator, 579213, 0, 0, 0, 0
l1cache, PutM_recv, , Accumulator, 579213, 0, 0, 0, 0
l1cache, PutE_recv, , Accumulator, 579213, 0, 0, 0, 0
l1cache, FetchInv_recv, , Accumulator, 579213, 0, 0, 0, 0
l1cache, FetchInvX_recv, , Accumulator, 579213, 0, 0, 0, 0
l1cache, Inv_recv, , Accumulator, 579213, 0, 0, 0, 0
l1cache, NACK_recv, , Accumulator, 579213, 0, 0, 0, 0
l1cache, MSHR_occupancy, , Accumulator, 579213, 0, 56, 56, 73
l1cache, evict_I, , Accumulator, 579213, 0, 0, 0, 0
l1cache, evict_E, , Accumulator, 579213, 0, 0, 0, 0
l1cache, evict_M, , Accumulator, 579213, 0, 0, 0, 0
l1cache, evict_IS, , Accumulator, 579213, 0, 0, 0, 0
l1cache, evict_IM, , Accumulator, 579213, 0, 0, 0, 0
l1cache, evict_IB, , Accumulator, 579213, 0, 0, 0, 0
l1cache, evict_SB, , Accumulator, 579213, 0, 0, 0, 0
l1cache, latency_GetS_IS, , Accumulator, 579213, 0, 56, 1568, 2
l1cache, latency_GetS_M, , Accumulator, 579213, 0, 0, 0, 0
l1cache, latency_GetX_IM, , Accumulator, 579213, 0, 0, 0, 0
l1cache, latency_GetX_SM, , Accumulator, 579213, 0, 0, 0, 0
l1cache, latency_GetX_M, , Accumulator, 579213, 0, 0, 0, 0
l1cache, latency_GetSX_IM, , Accumulator, 579213, 0, 0, 0, 0
l1cache, latency_GetSX_SM, , Accumulator, 579213, 0, 0, 0, 0
l1cache, latency_GetSX_M, , Accumulator, 579213, 0, 0, 0, 0
l1cache, EventStalledForLockedCacheline, , Accumulator, 579213, 0, 0, 0, 0
l1cache, evict_S, , Accumulator, 579213, 0, 0, 0, 0
l1cache, evict_SM, , Accumulator, 579213, 0, 0, 0, 0
l1cache, stateEvent_GetS_I, , Accumulator, 579213, 0, 2, 2, 2
l1cache, stateEvent_GetS_S, , Accumulator, 579213, 0, 0, 0, 0
l1cache, stateEvent_GetS_M, , Accumulator, 579213, 0, 0, 0, 0
l1cache, stateEvent_GetX_I, , Accumulator, 579213, 0, 0, 0, 0
l1cache, stateEvent_GetX_S, , Accumulator, 579213, 0, 0, 0, 0
l1cache, stateEvent_GetX_M, , Accumulator, 579213, 0, 0, 0, 0
l1cache, stateEvent_GetSX_I, , Accumulator, 579213, 0, 0, 0, 0
l1cache, stateEvent_GetSX_S, , Accumulator, 579213, 0, 0, 0, 0
l1cache, stateEvent_GetSX_M, , Accumulator, 579213, 0, 0, 0, 0
l1cache, stateEvent_GetSResp_IS, , Accumulator, 579213, 0, 0, 0, 0
l1cache, stateEvent_GetXResp_IS, , Accumulator, 579213, 0, 2, 2, 2
l1cache, stateEvent_GetXResp_IM, , Accumulator, 579213, 0, 0, 0, 0
l1cache, stateEvent_GetXResp_SM, , Accumulator, 579213, 0, 0, 0, 0
l1cache, stateEvent_Inv_I, , Accumulator, 579213, 0, 0, 0, 0
l1cache, stateEvent_Inv_S, , Accumulator, 579213, 0, 0, 0, 0
l1cache, stateEvent_Inv_IS, , Accumulator, 579213, 0, 0, 0, 0
l1cache, stateEvent_Inv_IM, , Accumulator, 579213, 0, 0, 0, 0
l1cache, stateEvent_Inv_SM, , Accumulator, 579213, 0, 0, 0, 0
l1cache, stateEvent_Inv_SB, , Accumulator, 579213, 0, 0, 0, 0
l1cache, stateEvent_Inv_IB, , Accumulator, 579213, 0, 0, 0, 0
l1cache, stateEvent_FetchInvX_I, , Accumulator, 579213, 0, 0, 0, 0
l1cache, stateEvent_FetchInvX_M, , Accumulator, 579213, 0, 0, 0, 0
l1cache, stateEvent_FetchInvX_IS, , Accumulator, 579213, 0, 0, 0, 0
l1cache, stateEvent_FetchInvX_IM, , Accumulator, 579213, 0, 0, 0, 0
l1cache, stateEvent_FetchInvX_SB, , Accumulator, 579213, 0, 0, 0, 0
l1cache, stateEvent_FetchInvX_IB, , Accumulator, 579213, 0, 0, 0, 0
l1cache, stateEvent_Fetch_I, , Accumulator, 579213, 0, 0, 0, 0
l1cache, stateEvent_Fetch_S, , Accumulator, 579213, 0, 0, 0, 0
l1cache, stateEvent_Fetch_IS, , Accumulator, 579213, 0, 0, 0, 0
l1cache, stateEvent_Fetch_IM, , Accumulator, 579213, 0, 0, 0, 0
l1cache, stateEvent_Fetch_SM, , Accumulator, 579213, 0, 0, 0, 0
l1cache, stateEvent_Fetch_IB, , Accumulator, 579213, 0, 0, 0, 0
l1cache, stateEvent_Fetch_SB, , Accumulator, 579213, 0, 0, 0, 0
l1cache, stateEvent_FetchInv_I, , Accumulator, 579213, 0, 0, 0, 0
l1cache, stateEvent_FetchInv_S, , Accumulator, 579213, 0, 0, 0, 0
l1cache, stateEvent_FetchInv_M, , Accumulator, 579213, 0, 0, 0, 0
l1cache, stateEvent_FetchInv_IS, , Accumulator, 579213, 0, 0, 0, 0
l1cache, stateEvent_FetchInv_IM, , Accumulator, 579213, 0, 0, 0, 0
l1cache, stateEvent_FetchInv_SM, , Accumulator, 579213, 0, 0, 0, 0
l1cache, stateEvent_FetchInv_SB, , Accumulator, 579213, 0, 0, 0, 0
l1cache, stateEvent_FetchInv_IB, , Accumulator, 579213, 0, 0, 0, 0
l1cache, stateEvent_FlushLine_I, , Accumulator, 579213, 0, 0, 0, 0
l1cache, stateEvent_FlushLine_S, , Accumulator, 579213, 0, 0, 0, 0
l1cache, stateEvent_FlushLine_M, , Accumulator, 579213, 0, 0, 0, 0
l1cache, stateEvent_FlushLine_IS, , Accumulator, 579213, 0, 0, 0, 0
l1cache, stateEvent_FlushLine_IM, , Accumulator, 579213, 0, 0, 0, 0
l1cache, stateEvent_FlushLine_SM, , Accumulator, 579213, 0, 0, 0, 0
l1cache, stateEvent_FlushLine_IB, , Accumulator, 579213, 0, 0, 0, 0
l1cache, stateEvent_FlushLine_SB, , Accumulator, 579213, 0, 0, 0, 0
l1cache, stateEvent_FlushLineInv_I, , Accumulator, 579213, 0, 0, 0, 0
l1cache, stateEvent_FlushLineInv_S, , Accumulator, 579213, 0, 0, 0, 0
l1cache, stateEvent_FlushLineInv_M, , Accumulator, 579213, 0, 0, 0, 0
l1cache, stateEvent_FlushLineInv_IS, , Accumulator, 579213, 0, 0, 0, 0
l1cache, stateEvent_FlushLineInv_IM, , Accumulator, 579213, 0, 0, 0, 0
l1cache, stateEvent_FlushLineInv_SM, , Accumulator, 579213, 0, 0, 0, 0
l1cache, stateEvent_FlushLineInv_IB, , Accumulator, 579213, 0, 0, 0, 0
l1cache, stateEvent_FlushLineInv_SB, , Accumulator, 579213, 0, 0, 0, 0
l1cache, stateEvent_FlushLineResp_I, , Accumulator, 579213, 0, 0, 0, 0
l1cache, stateEvent_FlushLineResp_IB, , Accumulator, 579213, 0, 0, 0, 0
l1cache, stateEvent_FlushLineResp_SB, , Accumulator, 579213, 0, 0, 0, 0
l1cache, eventSent_GetS, , Accumulator, 579213, 0, 2, 2, 2
l1cache, eventSent_GetX, , Accumulator, 579213, 0, 0, 0, 0
l1cache, eventSent_GetSX, , Accumulator, 579213, 0, 0, 0, 0
l1cache, eventSent_PutM, , Accumulator, 579213, 0, 0, 0, 0
l1cache, eventSent_NACK_down, , Accumulator, 579213, 0, 0, 0, 0
l1cache, eventSent_FlushLine, , Accumulator, 579213, 0, 0, 0, 0
l1cache, eventSent_FlushLineInv, , Accumulator, 579213, 0, 0, 0, 0
l1cache, eventSent_FetchResp, , Accumulator, 579213, 0, 0, 0, 0
l1cache, eventSent_FetchXResp, , Accumulator, 579213, 0, 0, 0, 0
l1cache, eventSent_AckInv, , Accumulator, 579213, 0, 0, 0, 0
l1cache, eventSent_GetSResp, , Accumulator, 579213, 0, 3, 3, 3
l1cache, eventSent_GetXResp, , Accumulator, 579213, 0, 0, 0, 0
l1cache, eventSent_FlushLineResp, , Accumulator, 579213, 0, 0, 0, 0
l1cache, stateEvent_AckPut_I, , Accumulator, 579213, 0, 0, 0, 0
l1cache, eventSent_PutS, , Accumulator, 579213, 0, 0, 0, 0
l1cache, eventSent_PutE, , Accumulator, 579213, 0, 0, 0, 0
l1cache, stateEvent_GetS_E, , Accumulator, 579213, 0, 2, 2, 2
l1cache, stateEvent_GetX_E, , Accumulator, 579213, 0, 0, 0, 0
l1cache, stateEvent_GetSX_E, , Accumulator, 579213, 0, 0, 0, 0
l1cache, stateEvent_FlushLine_E, , Accumulator, 579213, 0, 0, 0, 0
l1cache, stateEvent_FlushLineInv_E, , Accumulator, 579213, 0, 0, 0, 0
l1cache, stateEvent_FetchInv_E, , Accumulator, 579213, 0, 0, 0, 0
l1cache, stateEvent_FetchInvX_E, , Accumulator, 579213, 0, 0, 0, 0
memory, requests_received_GetS, , Accumulator, 579213, 0, 2, 2, 2
memory, requests_received_GetSX, , Accumulator, 579213, 0, 0, 0, 0
memory, requests_received_GetX, , Accumulator, 579213, 0, 0, 0, 0
memory, requests_received_PutM, , Accumulator, 579213, 0, 0, 0, 0
memory, outstanding_requests, , Accumulator, 579213, 0, 22, 22, 579
memory, latency_GetS, , Accumulator, 579213, 0, 22, 242, 2
memory, latency_GetSX, , Accumulator, 579213, 0, 0, 0, 0
memory, latency_GetX, , Accumulator, 579213, 0, 0, 0, 0
memory, latency_PutM, , Accumulator, 579213, 0, 0, 0, 0
memory, cycles_with_issue, , Accumulator, 579213, 0, 2, 2, 2
memory, cycles_attempted_issue_but_rejected, , Accumulator, 579213, 0, 0, 0, 0
memory, total_cycles, , Accumulator, 579213, 0, 579, 335241, 1
