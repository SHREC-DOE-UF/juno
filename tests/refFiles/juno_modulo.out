--------------------------------------------------------------------------
WARNING: a request was made to bind a process. While the system
supports binding the process itself, at least one node does NOT
support binding memory to the process location.

  Node:  sst-test

This usually is due to not having the required NUMA support installed
on the node. In some Linux distributions, the required support is
contained in the libnumactl and libnumactl-devel packages.
This is a warning only; your job will continue, though performance may be degraded.
--------------------------------------------------------------------------
Juno[cpu:JunoCPU:0]: Loading memory interface: memHierarchy.memInterface ...
Juno[cpu:JunoCPU:0]: Successfully loaded memory interface.
Juno[cpu:JunoCPU:0]: Opening program ../asm/modulo.bin ...
Juno[cpu:JunoProgramReader:0]: Application Version:      1000000
Juno[cpu:JunoProgramReader:0]: Data Length:              24
Juno[cpu:JunoProgramReader:0]: Inst Length:              48
Juno[cpu:JunoProgramReader:0]: Padding                   0
Juno[cpu:JunoCPU:0]: Creating an instruction manager...
Juno[cpu:JunoCPU:0]: PC set to: 24
Juno[cpu:JunoCPU:0]: Creating a register file of 16 64-bit integer registers...
Juno[cpu:JunoCPU:0]: Creating load/store unit...
Juno[cpu:JunoCPU:0]: Loading custom instructions...
Juno[cpu:JunoCPU:0]: Loaded 1 custom instruction handlers.
Juno[cpu:JunoCPU:0]: Loading operation cycle counts...
Juno[cpu:JunoCPU:0]: Configuring statistics...
Juno[cpu:JunoCPU:0]: Initialization done.
Juno[cpu:init:0]: Sending initialization data to memory...
Juno[cpu:init:0]: Initialization data sent.
Simulation is complete, simulated time: 579.213 ns
